nifi-nar-bundles/nifi-redis-bundle/nifi-redis-extensions/pom.xml

```diff
@@ -47,7 +47,7 @@
         <dependency>
             <groupId>redis.clients</groupId>
             <artifactId>jedis</artifactId>
-            <version>2.9.0</version>
```

Comment:
Is the `2.9.3` driver compatible and does it have that jedis fix in it? I tried to follow the issue but couldn't find the relevant commit.
If not, is 3.3.0 backwards compatible to a 2.9.0 Redis instance? And is there a reason your chose 3.3.0 versus the latest (3.5.1)?

Reply:
Yes, `2.9.3` has the fix in it as well and should work as well. 

I chose `3.3.0` only because it was the direct dependency of the latest version of spring-data-redis. As far as I can see in the changelog (https://docs.spring.io/spring-data/data-redis/docs/2.3.4.RELEASE/reference/html/#new-features), spring-data-redis has no breaking changes to existing APIs. As jedis is only used via spring-data-redis, I wanted to make sure that there are no problems introduced by incompatibilities with newer jedis versions.